### PR TITLE
Update print.h

### PR DESCRIPTION
**Pull request by mhsaar**
_Please use the following template to help us managing pull requests._

## Summary of Changes

Adding includes for what is used to allow header to be compiled on its own.

## Release Management

* Affected package(s):
* Issue(s) solved (if any): fix #0000, fix #0000,...
* Feature/Small Feature (if any):
* Link to compiled documentation (obligatory for small feature) [*wrong link name to be changed*](httpssss://wrong_URL_to_be_changed/Manual/Pkg)
* License and copyright ownership:

### CODE REVIEW

File: Straight_skeleton_2/test/Straight_skeleton_2/print.h [hunk 1]
```diff
@@ -1,5 +1,7 @@
-#include <CGAL/Polygon_2.h>
-#include <CGAL/Polygon_with_holes_2.h>
+#include <CGAL/Point_2>
```
Comment (sloriot) [inline Straight_skeleton_2/test/Straight_skeleton_2/print.h:1]:
```suggestion
#include <CGAL/Point_2.h>
```

File: Straight_skeleton_2/test/Straight_skeleton_2/print.h [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+#include <CGAL/Polygon_2>
```
Comment (sloriot) [inline Straight_skeleton_2/test/Straight_skeleton_2/print.h:2]:
```suggestion
#include <CGAL/Polygon_2.h>
```

File: Straight_skeleton_2/test/Straight_skeleton_2/print.h [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
+#include <CGAL/Polygon_with_holes_2>
```
Comment (sloriot) [inline Straight_skeleton_2/test/Straight_skeleton_2/print.h:3]:
```suggestion
#include <CGAL/Polygon_with_holes_2.h>
```

File: Straight_skeleton_2/test/Straight_skeleton_2/print.h [hunk 4]
```diff
@@ -1,5 +1,7 @@
...
+#include <CGAL/Straight_skeleton_2>
```
Comment (sloriot) [inline Straight_skeleton_2/test/Straight_skeleton_2/print.h:4]:
```suggestion
#include <CGAL/Straight_skeleton_2.h>
```

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.2-Ic-50.shtml

**Pull request merged**